lib/active_shipping/carriers/canada_post_pws.rb

```diff
@@ -294,6 +294,14 @@ def parse_rates_response(response, origin, destination)
       CPPWSRateResponse.new(true, "", {}, :rates => rates)
     end
 
+    def price_from_node(node, exclude_tax)
+      price = node.at('price-details/due').text
+      return price unless exclude_tax
+      children = node.at('price-details/taxes').children
+      tax_total_cents = children.sum { |node| node.elem? ? Package.cents_from(node.text) : 0 }
```

Comment:
Re-adding explanation since it was squashed

From some reason it thinks taxes has 7 children (counts newline in the xml response as a child). These newlines are of type `Text` where as the actual tax xml fields (gst,pst,hst) are of type `Element`, using `node.elem?` I skip over the newlines.